src/README.md

```diff
@@ -1,15 +1,95 @@
-The docs are built on GitHub Actions and run the ``.ci_scripts/update_docs`` script.
+The docs are built on GitHub Actions and run the `.ci_scripts/update_docs` script.
 
 To build the docs locally, follow the steps mentioned below:
- 1.  Clone the repository into your local machine.
- 2.  Go into the main folder and run the following commands.  
- 3. ``conda env create -f ./.ci_scripts/environment.yml``
- 4. ``conda activate conda-forge-docs``
- 5. ``cd newsfeed && pip install --no-deps .``
- 6. ``cd ../src``
- 7. ``make html``
 
-Documentation update PRs should only change the files in ``src``. 
+1.  Clone the repository into your local machine.
+2.  Go into the main folder and run the following commands.
 
-Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
+```
+conda env create -f ./.ci_scripts/environment.yml
+```
 
+```
+conda activate conda-forge-docs
+```
+
+```
+cd newsfeed && pip install --no-deps .
+```
+
+```
+cd ../src
+```
+
+```
+make html
+```
+
+# Contributor Conda-forge Code of Conduct
+
+## Scope
+
+This code of conduct applies to all spaces managed by Conda-forge, including all public and private mailing lists, issue trackers, wikis, forums, and any other communication channel used by our community. The code of conduct equally applies at Conda-forge events and governs standards of behavior for attendees, speakers, volunteers, booth staff, and event sponsors.
+
+## Our Dedication
+
+Conda-forge is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+- Be kind to others.
+- Do not insult or put down others.
+- Behave professionally.
+- Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for conda-forge.
+- All communication should be appropriate for a professional audience including people of many different backgrounds.
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery.
+- Violent or intimidating threats or language directed against another person.
+- Trolling or insulting and derogatory comments.
+- Public or private harassment.
+- Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent.
+- Posting sexually explicit or violent material.
+- Other unethical or unprofessional conduct
+- Advocating for, or encouraging, any of the above behaviors
+
+## Reporting a Misconduct
+
+- You can reach the Project maintainers.
```

Comment:
If you're including this section, you'll need to put a way to reach the conda-forge maintainers. You can link to the https://conda-forge.org/docs/orga/getting-in-touch.html page and then people can decide how they want to communicate an issue from there.